packages/util/README.md

```diff
@@ -80,6 +80,8 @@ import { stripHexPrefix } from 'ethereumjs-util'
 
 ### Re-Exports
 
+(Please note these are deprecated and planned to be removed in the next major release, util v8)
```

Comment:
I am not sure if we can say this so confidentaly on the `BN.js` export, I guess this only holds if we really manage to remove `BN.js` completely from our monorepo?

So I would leave this open for now.

In case we do not remove we likely need this re-export to not fall into this trap again of eventually having different BN.js major version BNs being passed around between the libraries and not being compatible with each other? 🤔 Eventually this is not relevant though if we are really determined to have this current BN version as the last version we support/have integrated. Not sure though if we can say this with confidence right now.

Reply:
sounds good, will remove the deprecation notices